client.go

```diff
@@ -177,6 +181,28 @@ func newClient(opts clientOpts) (*client, error) {
 	}, nil
 }
 
+var cleanupFreq = 10 * time.Second
```

Comment:
Nit: can probably be on the order of minutes. But not really an issue any which way.